extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -497,6 +498,16 @@ describes.fakeWin('amp-analytics.VariableService', {amp: true}, (env) => {
       return check('CUMULATIVE_LAYOUT_SHIFT', '1');
     });
 
+    it('should expand EXPERIMENT_BRANCHES to name:value comma separated list', () => {
+      forceExperimentBranch(env.win, 'exp1', '1234');
+      forceExperimentBranch(env.win, 'exp2', '5678');
+      return check('EXPERIMENT_BRANCHES', 'exp1%3A1234%2Cexp2%3A5678');
```

Comment:
Thanks for including the experimentName. 

Might worth accepting experiment name to the macro so `EXPERIMENT_BRANCHES(exp1)` resolves to `1234`.

Reply:
Do you think this should be separate macro since the return value is different? One would return a list of kv pairs, the other a single id.

Reply:
If I were to design this from start. I would make the experiment name mandatory. So one would only be collecting relative experiment info. And as you suggested, it would save parsing effort because you can format the request however you like. 

separate macro: I think it's fine as long as we document it clearly.

Reply:
Ok added optional param for experiment name.